pkg/validation/validation_test.go

```diff
@@ -29,6 +29,16 @@ func TestValidationForKubernetes(t *testing.T) {
 		assert.Error(t, err)
 	})
 
+	t.Run("invalid length if suffix -dapr is appended", func(t *testing.T) {
+		id := ""
```

Comment:
Just use a const here is enough, no need to use loop.

Reply:
I replaced the loop with a single value in 4f5af0f6e40d6dc2b78536a09ad2e864545f0fe5.
`strings.Repeat` is used for length readability.